packages/extension-sample/src/setup-app.ts

```diff
@@ -7,16 +7,19 @@
 
 import React from 'react'
 import loadable from '@loadable/component'
-import {IRouteConfig} from '@salesforce/pwa-kit-react-sdk/ssr/universal/extensibility/types'
-import {ApplicationExtension} from '@salesforce/pwa-kit-react-sdk/ssr/universal/extensibility'
+import {ApplicationExtension, ApplicationExtensionConfig, IRouteConfig} from '@salesforce/pwa-kit-react-sdk/ssr/universal/extensibility'
 
 import withRedBorder from './components/with-red-border'
 
 const SamplePage = loadable(() => import('./pages/sample'))
-
 const defaultPath: string = '/sample-page'
-class Sample extends ApplicationExtension {
-    
+
+interface Config extends ApplicationExtensionConfig {
+    // react-router-style path to the new sample page
+    path?: string
+}
```

Comment:
Inside the Application Extension project folder there is a file called `src/types/config.ts`. Can we move this type into that file. I think it will add to our extension "template" and give guidance to developers on where to put their types when building their projects. 

Even tho they can technically put them anywhere they please, it's nice to organize it a little.